Paco-Server/src/com/google/sampling/experiential/server/EnvironmentUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sampling.experiential.server;
+
+public final class EnvironmentUtil {
+  private static final String RUN_TIME_VERSION_PROPERTY = "com.google.appengine.runtime.version";
+  private static final String DEV_HEADER = "Google App Engine/";
+  
+  private EnvironmentUtil() {
+
+  }
+ 
+  public static final boolean isDevServer(){
+    return !System.getProperty(RUN_TIME_VERSION_PROPERTY).startsWith(DEV_HEADER);
```

Comment:
The Appengine documentation (https://cloud.google.com/appengine/docs/standard/java/tools/using-local-server) uses the following method to test for development or prod mode:

  public static boolean isDevInstance(HttpServletRequest req) {
    return SystemProperty.environment.value() == SystemProperty.Environment.Value.Development;
  }

Also, we have an instance of this method in AuthUtil. We should unify them into this new util you have created.

Reply:
I changed it to the one you specified. It sounds like a correct property name to rely on. I got it from the cloud sql code sample. https://cloud.google.com/appengine/docs/standard/java/cloud-sql/

Reply:
Funny. I guess either one would work then...